sdk/ai/azure-ai-inference/pyproject.toml

```diff
@@ -1,2 +1,5 @@
+[tool.azure-sdk-build]
+strict_sphinx = true
+
```

Comment:
You can remove this (it's already removed main). As of yesterday, sphinx == strict_sphinx and sphinx is now defaulted true so we don't need to add it to the pyproject.toml anymore

Reply:
Removed! thanks @kristapratico